ethcore/res/ethereum/classic.json

```diff
@@ -3077,7 +3077,9 @@
 		"enode://72e445f4e89c0f476d404bc40478b0df83a5b500d2d2e850e08eb1af0cd464ab86db6160d0fde64bd77d5f0d33507ae19035671b3c74fec126d6e28787669740@104.198.71.200:30303",
 		"enode://39abab9d2a41f53298c0c9dc6bbca57b0840c3ba9dccf42aa27316addc1b7e56ade32a0a9f7f52d6c5db4fe74d8824bcedfeaecf1a4e533cacb71cf8100a9442@144.76.238.49:30303",
 		"enode://f50e675a34f471af2438b921914b5f06499c7438f3146f6b8936f1faeb50b8a91d0d0c24fb05a66f05865cd58c24da3e664d0def806172ddd0d4c5bdbf37747e@144.76.238.49:30306",
-		"enode://83b33409349ffa25e150555f7b4f8deebc68f3d34d782129dc3c8ba07b880c209310a4191e1725f2f6bef59bce9452d821111eaa786deab08a7e6551fca41f4f@159.89.223.6:30303"
+		"enode://83b33409349ffa25e150555f7b4f8deebc68f3d34d782129dc3c8ba07b880c209310a4191e1725f2f6bef59bce9452d821111eaa786deab08a7e6551fca41f4f@159.89.223.6:30303",
+		"enode://5cd218959f8263bc3721d7789070806b0adff1a0ed3f95ec886fb469f9362c7507e3b32b256550b9a7964a23a938e8d42d45a0c34b332bfebc54b29081e83b93@35.187.57.94:30303",
+		"enode://6dd3ac8147fa82e46837ec8c3223d69ac24bcdbab04b036a3705c14f3a02e968f7f1adfcdb002aacec2db46e625c04bf8b5a1f85bb2d40a479b3cc9d45a444af@104.237.131.102:30303"
```

Comment:
why is this pr changing bootnode addresses of unrelated chains?

Reply:
@debris Cause this is what we call open source, I can revert the commit but as you know there is no fixed maintainer of these chains, all bootnodes are being updated from parity team or volunteers like me.

I've updated the list based on these list from their chain client, you can refer them here

ETC: https://raw.githubusercontent.com/ethereumproject/go-ethereum/master/core/config/mainnet_bootnodes.json

Ropsten: https://raw.githubusercontent.com/ethereumproject/go-ethereum/master/core/config/morden_bootnodes.json

ELLA: https://raw.githubusercontent.com/ellaism/parity-config/master/ellaism.json

EXP: https://raw.githubusercontent.com/expanse-org/go-expanse/master/params/bootnodes.go

MUSIC: https://raw.githubusercontent.com/Musicoin/go-musicoin/master/params/bootnodes.go

Reply:
@debris So should I revert the commit and seperate the PR??

Reply:
It would be great 🙌

Reply:
This will allow us to easily backport bootnodes update to stable and beta

Reply:
@debris Seperated the PR for bootnode update 😄